python/python/tests/test_async_client.py

```diff
@@ -1664,6 +1668,244 @@ async def test_geoadd_invalid_args(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.geoadd(key, {"Place": GeospatialData(0, -86)})
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_geosearch_by_box(self, redis_client: TRedisClient):
+        key = get_random_string(10)
+        members = ["Catania", "Palermo", "edge2", "edge1"]
+        members_coordinates = {
```

Comment:
It is possible to find in the internet more data and search examples for testing?
On another hand, we don't test redis, we test the client.

Reply:
I think one example is enough

Reply:
If you have more examples in your notes - please post them to a wiki page or keep them to post later.

Reply:
I dont, but I guess we can create some